vmm/src/lib.rs

```diff
@@ -1444,6 +1444,15 @@ impl Vmm {
                     VmConfigError::InvalidVcpuCount,
                 ));
             }
+            #[cfg(target_arch = "x86_64")]
```

Comment:
This check is already enforced because setup_mptable is called as part of the `InstanceStart` routine. Since you also add a check for vcpu_count < 32 in the following commit, I think this one should be removed. What do you say?

Reply:
This arch specific code should be removed from the generic part, done.